### PR TITLE
Adds more dice and put them into the custom loadout

### PR DESCRIPTION
**Pull request by Alberyk**
Adds more dice and give the dice bags some new sprites, also put them in the loadout since people asked on the forums.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 1]
```diff
@@ -3,8 +3,12 @@
 	path = /obj/item/weapon/cane
 
 /datum/gear/dice
-	display_name = "d20"
-	path = /obj/item/weapon/dice/d20
+	display_name = "pack of dices"
```
Comment (Lohikar) [inline code/modules/client/preference_setup/loadout/loadout_general.dm]: This should just be "dice".

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 2]
```diff
@@ -3,8 +3,12 @@
...
+	path = /obj/item/weapon/storage/pill_bottle/dice
+
+/datum/gear/dicegaming
+	display_name = "pack of gaming dices"
```
Comment (Lohikar) [inline code/modules/client/preference_setup/loadout/loadout_general.dm]: same here

File: code/modules/client/preference_setup/loadout/loadout_general.dm (hunk 1 above)
Comment (Alberyk) [reply code/modules/client/preference_setup/loadout/loadout_general.dm]: It is more than one.

Comment (Lohikar) [reply code/modules/client/preference_setup/loadout/loadout_general.dm]: `dice` is plural. `dices` is not a word.

**Pull request merged**